go.mod

```diff
@@ -0,0 +1,8 @@
+module pokedex
```

Comment:
Your module name should be something related to where your project is hosted.

In your case, it should be something like:
``` suggest
module github.com/hugoaguirre/pokedex
```
or something similar.